src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -1045,6 +1044,8 @@ bool Creature::Create(ObjectGuid::LowType guidlow, Map* map, uint32 phaseMask, u
     if (!CreateFromProto(guidlow, entry, data, vehId))
         return false;
 
+    UpdatePositionData();
```

Comment:
what is this change for ?

Reply:
If I call it earlier assertion will trigger in GetColllisionHeight, or crash in GetUInt32Value. Can't remember which one it was.

Reply:
Seems like https://github.com/TrinityCore/TrinityCore/issues/20028 is back now with these changes so, I will maybe revert those changes and let the default param do the job